app/messaging/back-office/process-backoffice.js

```diff
@@ -1,27 +1,24 @@
-const util = require('util')
-const { getAll, getApplicationCount } = require('../repositories/application-repository')
-const { backOfficeResponseMsgType, backOfficeResponseQueue } = require('../config')
-const sendMessage = require('../messaging/send-message')
-const states = require('./states')
+const { getAll, getApplicationCount } = require('../../repositories/application-repository')
+const { backOfficeResponseMsgType, backOfficeResponseQueue } = require('../../config')
+const sendMessage = require('../../messaging/send-message')
+const states = require('../states')
 
 const processBackOfficeRequest = async (msg) => {
   const responseMessage = msg.body
   const { sessionId } = msg
   try {
-    console.log('BackOffice Request received:', util.inspect(msg.body, false, null, true))
-    // Get ID
-    const result = await getAll(msg.body.limit ?? 10, msg.body.offset ?? 0, msg.body.search.text)
+    // Get All Applications
```

Comment:
The comment doesn't add any value, if you want to make it clear this getting all applications consider renaming the function below.